forui/lib/src/widgets/card/card_style.dart

```diff
@@ -1,21 +1,44 @@
 part of 'card.dart';
 
 /// [FCard]'s style.
-class FCardStyle {
+final class FCardStyle with Diagnosticable {
   /// The decoration.
   final BoxDecoration decoration;
 
+  /// The body.
+  final TextStyle text;
+
```

Comment:
Where is this used?

Reply:
This is used in https://github.com/forus-labs/forui/blob/9154bd8e85fef5fdd68b40edef3cbd7f89424a48/forui/lib/src/widgets/card/card.dart#L36.

Reply:
I see, if that's the case we should remove this. It's largely pointless imo and might result in unexpected TextStyle. I'd assume most users will think a `Text` will behave the same anywhere throughout the application.